drools-wb-screens/drools-wb-test-scenario-editor/drools-wb-test-scenario-editor-backend/src/test/java/org/drools/workbench/screens/testscenario/backend/server/ScenarioTestEditorServiceImplTest.java

```diff
@@ -314,6 +314,37 @@ public void testCopyWithTargetThrowException() throws Exception {
     }
 
     @Test
+    public void testRunScenarioDependentImportsAndFactData() throws Exception {
+        final ArrayList<Fixture> fixtures = new ArrayList<Fixture>();
+        final Imports imports = new Imports() {{
+            addImport(new Import("java.sql.ClientInfoStatus"));
+            addImport(new Import("Integer"));
+            addImport(new Import("org.kie.Person"));
+        }};
+
+        final Map<String, ModelField[]> modelFields = new HashMap<String, ModelField[]>() {{
+            put("java.sql.ClientInfoStatus",
+                new ModelField[]{modelField("java.sql.JDBCType")});
```

Comment:
Looks like boilerplate.

Consider introducing `static ModelField[] toModelFields(String... m)` to hide the complexity.

Reply:
Will apply.